lib/beeleex_web/endpoint.ex

```diff
@@ -0,0 +1,43 @@
+defmodule BeeleexWeb.Endpoint do
```

Comment:
This is not necessary as this is a library that doesn't need to run its own endpoint